microservices/forumApi/routes/v1/collaborators/collaborators.js

```diff
@@ -0,0 +1,36 @@
+var db = require('../db/db');
+
+var collaborators = {};
+
+collaborators.subscribe = function(topicId, userId, callback){
+    db.Topic.findById(topicId).exec((err, topic) => {
+        if (err || topic == null) {
+            callback({error:'topic not found'});
+        } else if (topic.contributors.includes (userId)) {
+            callback();
+        } else {
+            topic.contributors.push (userId);
```

Comment:
should not be editing contributors, as subscription is not a contributor, should be a new field